LSICORR/iocBoot/iocLSICORR-IOC-01/runIOC.bat

```diff
@@ -0,0 +1,13 @@
+@echo off
+setlocal
+set MYDIRBLOCK=%~dp0
+call C:\Instrument\Apps\EPICS\config_env_base.bat
+
+%HIDEWINDOW% h
+
+set EPICS_CAS_INTF_ADDR_LIST=127.0.0.1
+set EPICS_CAS_BEACON_ADDR_LIST=127.255.255.255
+
+set PYTHONUNBUFFERED=TRUE
+
+%PYTHON3W% %EPICS_KIT_ROOT%/support/lsicorr/master/LSi_Correlator.py --pv_prefix %MYPVPREFIX% --ioc_number 1
```

Comment:
Because twisted is not compatible with python 3.8 yet, your default genie_python will no longer work. I recommend stealing the deployed version of python 3 from zoom which has the right dependencies, and replacing %PYTHON3W% with the new location

Reply:
Can you put a note on the developer wiki about this? Also mention what we decided to do about and who decided that.

Reply:
Sure: https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/LSi-Correlator#twisted-and-python-38